Sources/SwiftGenKit/Parsers/Strings/FileTypeParser/StringsFileTypeParser.swift

```diff
@@ -0,0 +1,17 @@
+//
+// SwiftGenKit
+// Copyright © 2019 SwiftGen
+// MIT Licence
+//
+
+import Foundation
+import PathKit
+
+protocol StringsFileTypeParser: AnyObject {
+  init(options: ParserOptionValues)
+
+  static var priority: Int { get }
```

Comment:
Since `priority` without any context isn't really telling, I'd suggest to add a doc comment.

```suggestion
  /// Parsers with a higher-priority will replace identical keys from lower-priority parsers.
  static var priority: Int { get }
```

Reply:
Good idea!

Reply:
Actually almost made that same comment on the `StringsFileParser.swift` and `StringsDictFileParser.swift` files before realising you already thought of it and documented it in the protocol 👍  Great minds… 😉